utils/build/docker/dotnet_http/parametric/.dockerignore

```diff
@@ -0,0 +1,30 @@
+**/.classpath
```

Comment:
Where did this come from? Why not copy the previous one from https://github.com/DataDog/system-tests/blob/main/utils/build/docker/dotnet/parametric/.dockerignore ?

Reply:
That one was auto generated, will change it now!